### PR TITLE
Added Lenovo Badge 

### PR DESCRIPTION
**Pull request by gurjeetsinghvirdee**

### CODE REVIEW

**Comment by weeyin83** [comment]
Great suggestion!

File: README.md [hunk 1]
```diff
@@ -66,6 +67,14 @@ Copy the appropriate `![Name](link)` element and paste it in your Markdown file
 [![](https://img.shields.io/badge/back%20to%20top-%E2%86%A9-blue)](#shortcuts)
 
 
+| Name             | Badge                                                                                                                                   | Markdown                                                                                                                                  |
```
Comment (Ileriayo) [inline README.md:70]: This is an orphaned section.
Also, try to keep your PR to a specific thing.

**Review by Ileriayo — CHANGES_REQUESTED** [review]
There's more than Lenovo badge in this PR with 38 additions and 27 deletions.

Can you create separate PRs for separate issues?

File: README.md (hunk 1 above)
Comment (gurjeetsinghvirdee) [reply README.md]: actually, it was a blog section

**Comment by gurjeetsinghvirdee** [comment]
@Ileriayo making new PR closing this one my system was lagging

**Pull request closed without merging**